tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -0,0 +1,166 @@
+
+2. Select **Role Collections** > **New Role Collection**.
+
+    ![Create role collection](2.2.create-role-collection.png)
+
+3. In the **Name** field, enter **`Tenant_Administrator`** and select **Save**. Select the newly created **`Tenant_Administrator`** role collection.
+
+    ![Create tenant admin role collection](2.3.create-tenant-admin-rc.png)
+
+4. Select **Add Role**. In the **Role Template** dropdown list, select **`AuthGroup_Administrator`** and select **Save**.
+
+    ![Add required roles](2.4.add-role.png)
+
+    Similarly, add **`AuthGroup_BusinessExpert`** and **`AuthGroup_IntegrationDeveloper`** role templates and add these to the role collection.
+
+[VALIDATE_7]
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Configure trust)]
+
+1. Select your subaccount name to navigate to your subaccount level. Select **Trust Configuration**.
+
+    ![Access trust configuration](3.1.access-trust-configuration.png)
+
+2. Select the default identity provider, **SAP ID Service**.
+
+    ![Select default IDP](3.2.select-default-idp.png)
+
+3. In the **E-Mail Address** field, enter your email ID and select **Show Assignments**. Since your user is not a part of the SAP ID Service, you get a prompt asking for the user to be added to SAP ID Service. Choose **Add User**.
+
+    ![Assign user to IDP](3.3.assign-user.png)
+
+4. Select **Show Assignments** > **Assign Role Collection**. In the **Assign Role Collection** prompt, choose **`Tenant_Administrator`** from the dropdown list and select **Assign Role Collection**.
+
+    ![Assign Tenant_Administrator](3.4.assign-role-collection.png)
+
+    These roles are required for you to create and model your integration flow.
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Create service instance and key)]
+1. Access your subaccount's space by choosing **Spaces** > **dev**. If you have intentionally specified any other space name, you can choose the space of your choice.
+
+    ![Access space](4.1.access-space.png)
+
+2. Choose **Service Marketplace** > **Process Integration Runtime**.
+
+    ![Access PI Runtime](4.2.access-process-integration-runtime.png)
+
+    >**TIP:** If you do not see the **Process Integration Runtime** service in the list, at the subaccount level, choose **Entitlements** > **Configure Entitlements** > **Add Service Plans**. Choose **Process Integration Runtime** and select the **integration-flow** service plan and choose **Add 1 Service Plan**.
+
+3. Select **Instances** > **New Instance**.
+
+    ![Create new instance](4.3.create.new.instance.png)
+
+4. Choose **Next** to select the default service plan. In the **Specify Parameters** tab, enter:
+
+    ```JSON
+    {
+     "roles":[
+       "ESBMessaging.send"
+     ]
+    }
+    ```
+
+    ![Specify JSON for user role](4.4.specify-json-instance.png)
+
+5. Choose **Next** until you have to specify the instance name. Provide any name of your choice and choose **Finish**.
```

Comment:
I am also stuck at this step (I get an error and no instance is created). If this is unstable on trial and there is not way to fix this by Sept 30th we shouldn't feature this scenario.